website_sale_hide_price/data/product_snippet_template_data.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
```

Comment:
When configuring prices to be hidden at the website level in settings, it has no effect on the product carousel. I am not sure which of these snippets needs to be adjusted or if any more need to be added.

![image](https://github.com/user-attachments/assets/cc54a9a0-2c20-4a70-a9d0-c79387ea27c7)